src/main/java/uk/gov/companieshouse/web/accounts/controller/cic/CompanyActivitiesAndImpactController.java

```diff
@@ -0,0 +1,91 @@
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.web.accounts.annotation.NextController;
+import uk.gov.companieshouse.web.accounts.annotation.PreviousController;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.controller.smallfull.ReviewController;
+import uk.gov.companieshouse.web.accounts.controller.smallfull.StatementsController;
+import uk.gov.companieshouse.web.accounts.controller.smallfull.StepsToCompleteController;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.cic.statements.CompanyActivitiesAndImpact;
+import uk.gov.companieshouse.web.accounts.service.cic.statements.CompanyActivitiesAndImpactService;
+import uk.gov.companieshouse.web.accounts.validation.ValidationError;
+
+
+@Controller
+@NextController(StepsToCompleteController.class)
+@PreviousController(CICStepsToCompleteController.class)
+@RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/cic/company-activity")
+public class CompanyActivitiesAndImpactController extends BaseController {
+
+    private static final UriTemplate CONFIRMATION_REDIRECT = new UriTemplate(
+        "/transaction/{transactionId}/confirmation");
+
+    @Autowired
+    private CompanyActivitiesAndImpactService companyActivitiesAndImpactService;
+
+    @Override
+    protected String getTemplateName() {
+        return "cic/companyActivitiesAndImpact";
+    }
+
+    @GetMapping
+    public String getCompanyActivitiesAndImpact(@PathVariable String companyNumber,
+        @PathVariable String transactionId,
+        @PathVariable String companyAccountsId,
+        Model model,
+        HttpServletRequest request) {
+
+        try {
+            model.addAttribute("companyActivitiesAndImpact", companyActivitiesAndImpactService.getCompanyActivitiesAndImpact(transactionId, companyAccountsId));
+        } catch (ServiceException e) {
+
+            LOGGER.errorRequest(request, e.getMessage(), e);
+            return ERROR_VIEW;
+        }
+
+        return getTemplateName();
+    }
+
+    @PostMapping
+    public String postCompanyActivitiesAndImpact(@PathVariable String companyNumber,
+        @PathVariable String transactionId,
+        @PathVariable String companyAccountsId,
+        @ModelAttribute("companyActivitiesAndImpact") @Valid CompanyActivitiesAndImpact companyActivitiesAndImpact,
+        BindingResult bindingResult,
+        Model model,
+        HttpServletRequest request) {
+
+        if (bindingResult.hasErrors()) {
```

Comment:
This won't ever be true; there's no field level validation enforced on the `CompanyActivitiesAndImpact` model so binding errors will never occur.